src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -60,6 +60,41 @@ describe('LSP3UniversalProfile', () => {
       expect(data[0].startsWith('0x6f357c6a')).toBe(true);
     });
   });
+  describe('Deploying with LSP3Profile Metadata with specified IPFS client options', () => {
+    let signer;
+    let universalProfile;
+    let keyManager;
+
+    beforeAll(async () => {
+      signer = signers[0];
+
+      const { ERC725Account, KeyManager } = (await lspFactory.LSP3UniversalProfile.deploy(
```

Comment:
Good point thanks. It could make sense to use ERC725Account if no profile data is passed. I think surrently if none is passed then the SupportedStandards:LSP3UniversalProfile key would be set even if there is no LSP3Profile data if I understand correctly

Reply:
Though this change is beyond scope of this PR so we can address that separately